### DISCUSSION TITLE
Zirkulationspumpe und Heizstab an VWZ AI

### DISCUSSION THREAD

**Discussion by MarkusGH** [cyberthom42/vaillant-arotherm-plus#19] [General]
Habe hier eine Anlage mit einer VWZ AI (Wärmepumpenregler anstelle Hydraulikstation)

Zirkulation und Heizstab werden über Multifunktionsausgänge angesteuert.
Hat eigentlich schon jemand hinbekommen, die Zirkulationspumpe anzusteuern?
Momentan behelfe ich mir mit Änderung des Zeitprogramms, aber das ist sehr unschön.

Heizstab dito - Kann man den irgendwie an-/abschalten?
Theoretisch ist der verwendete Heizstab zweistufig, aber irgendwie kriegt das das Regelungsmodul auch nicht hin.

**Comment by cyberthom42** [comment]
Bei mir läuft die Zirkulationspumpe über die Trinkwasserstation. Wenn sie bei dir über die Multifunktionseingänge geht, lassen sich diese doch prima steuern. Hast du mal versucht den MA1 zu schreiben? z.B.
`b514,057703ffff`

**Comment by MarkusGH** [comment]
Die Pumpe hängt am MA2, Zusatzheizung an ZH.

Die Pumpe ist an, aber ebusd zeigt mir das:

```
{
 "hmu": {
  "messages": {   "HE_MA2_Ausgang": {
    "name": "HE_MA2_Ausgang",
    "passive": false,
    "write": false,
    "lastup": 1694808992,
    "zz": 8, "comment": "T.1.126",
    "fields": {
     "str": {"value": "off", "comment": "MA2 Ausgang"}
    }
   }
  }
 },
```

**Reply by cyberthom42** [reply]
Naja, in der CSV steht halt das man nur lesen kann.
Kannst ja mal versuchen den Wert in der CSV schreibbar zu machen.

**Reply by MarkusGH** [reply]
Ist mir bewusst - aber HE_MA2_Ausgang scheint den Zustand des MA2 ja schon gar nicht korrekt wiederzuspiegeln.

**Reply by cyberthom42** [reply]
Der ist aktuell auf off, ist das nicht richtig? Der Ausgang wird ja sicherlich nur eingeschaltet, wenn die Pumpe laufen soll. Übrigens, da fällt mir noch ein, man kann auch am 720 die Eingänge konfigurieren und mit Zirkulation verknüpfen. Hattest du das mal probiert? Evtl. kann man ja den Wert über die 720 schreiben. 
Hab gerade gesehen, den habe ich gar nicht in der 720er CSV

**Reply by cyberthom42** [reply]
Was du machen könntest wäre ein HEX Scan. So habe ich unbekannte Werte gefunden. Scan vorher , Zirkulation einschalten (z.B. Zeitplan), Scan danach. Nun ein Diff um die Änderungen herauszufinden. Ist ein wenig mühselig :-) Bei mir wird die ja über die Trinkwasserstation gesteuert, daher kann ich da gar nicht mit einem Scan helfen

**Reply by MarkusGH** [reply]
Nein - die (geregelte) Pumpe läuft.
Habe einen 700er, keinen 720er.

**Reply by MarkusGH** [reply]
> Was du machen könntest wäre ein HEX Scan. 

Finde dazu nix - wie geht das?
